roles/pulp_webserver/templates/nginx.conf.j2

```diff
@@ -32,8 +31,30 @@ http {
 
     server {
         # Gunicorn docs suggest the use of the "deferred" directive on Linux.
+{% if pulp_webserver_disable_https | bool %}
         listen 80 default_server deferred;
         listen [::]:80 default_server deferred;
+{% else %}
+        listen 443 default_server deferred ssl;
+        listen [::]:443 default_server deferred ssl;
+
+        ssl_certificate {{ pulp_webserver_tls_folder }}/pulp_webserver.crt;
+        ssl_certificate_key {{ pulp_webserver_tls_folder }}/pulp_webserver.key;
+        ssl_session_cache shared:SSL:50m;
+        ssl_session_timeout 1d;
+        ssl_session_tickets off;
+
+        # intermediate configuration
+        ssl_protocols TLSv1.2;
+        ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+        ssl_prefer_server_ciphers on;
```

Comment:
Do we really want to specify these? If we do not, will the system use some (sane/secure) distro default or system-wide crypto policy? If so, that would be much preferable.